src/Form/InputRadio.php

```
@@ -0,0 +1,269 @@
+        ] + parent::loadDefault();
+    }
+
+    /**
+     * Renders the `<input>` element with its attributes.
+     *
+     * @return string Rendered HTML for the `<input>` element.
+     */
+    protected function run(): string
+    {
+        /** @var string|null $id */
+        $id = $this->getAttribute('id', null);
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $id. Configured minimum length is 3. (undefined)

(ShortVariable)